Formula/steem.rb

```diff
@@ -0,0 +1,33 @@
+class Steem < Formula
+  desc "Proof of Work blockchain with an unproven consensus algorithm."
+  homepage "https://steemit.com/"
+  url "https://github.com/steemit/steem.git",
+      :tag => "v0.8.5b",
```

Comment:
Might be worth having @TigerND ask the developer his preference. It makes me think of how Gmail called itself beta for five million years though :)